lib/AST/ASTMangler.cpp

```diff
@@ -1382,7 +1382,13 @@ void ASTMangler::appendType(Type type, GenericSignature sig,
                          /*autoclosure*/ false,
                          forDecl);
       return;
-      
+
+    case TypeKind::MoveOnly: {
+      // For now just append the inner type. TODO: What operator do I use here?
+      auto moveOnly = cast<MoveOnlyType>(tybase);
```

Comment:
@eeckstein @atrick  @jckarter I was not sure what to do here. I think I need to append an operator not just mangled in the type. What operator do you suggest I use?